spec/index.bs

```diff
@@ -813,14 +813,16 @@ dictionary GPUObjectDescriptorBase {
         The initial value of {{GPUObjectBase/label|GPUObjectBase.label}}.
 </dl>
 
-## Invalid Internal Objects &amp; Contagious Invalidity ## {#invalidity}
+## Asynchrony ## {#asynchrony}
 
-Object creation operations in WebGPU are internally asynchronous, so they don't fail with exceptions.
-Instead, returned objects may refer to [=internal objects=] which are either
+### Invalid Internal Objects &amp; Contagious Invalidity ### {#invalidity}
+
+Object creation operations in WebGPU don't return promises, but nonetheless are internally
+asynchronous, and don't fail with exceptions or rejections. Instead, returned objects refer to
+[=internal objects=], which exist on a [=device timeline=] where they are either
```

Comment:
This paragraph reads strangely to me. I'd suggest something along the lines of:

"Object creation operations in WebGPU don't return promises, but nonetheless are internally asynchronous. Returned objects refer to [=internal objects=] which are manipulated on a [=device timeline=]. Rather than fail with exceptions or rejections, most errors that occur on the a [=device timeline=] are communicated through {{GPUError}}s generated on the relevant [=device=].

[=Internal objects=] are either <dfn dfn>valid</dfn> or <dfn dfn>invalid</dfn>.  Objects that are [=invalid=]..."

Reply:
Much better!